pcap/pkt/pkt_unsafe.go

```diff
@@ -108,8 +134,14 @@ func (p *Packet) decode() {
 }
 
 type TcpPacket struct {
```

Comment:
Perhaps we should add an `isIPv6` flag to the struct definition for user-friendliness?